### PR TITLE
Deeplab py3

### PR DESCRIPTION
**Pull request by DefineFC**
Hope DeepLab model will soon support py3. I tried to modify a few files to run it on py3, currently tested data_preperation/train/eval for Cityscapes & Pascal.

> For Cityscapes, change `ciyscapesscripts/helpers/annotation.py`: near line 238 into `print(type(obj).__name__)` (just add the parentheses).  
> Someone has opened [a PR about that change](https://github.com/mcordts/cityscapesScripts/pull/47) for [cityscapesScripts](https://github.com/mcordts/cityscapesScripts), hope they accept it soon.

Any discussions will help.

A further step of #3741

### CODE REVIEW

**Comment by DefineFC** [comment]
just a different email address, what should I do to pass the cla check :-(

**Comment by DefineFC** [comment]
are we good here?

File: research/deeplab/datasets/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# datafiles
```
Comment (YknZhu) [inline research/deeplab/datasets/.gitignore]: please remove this file.

Comment (DefineFC) [reply research/deeplab/datasets/.gitignore]: Removed :-) thought it would help contribute.

**Pull request merged**